dev/make-distribution.sh

```diff
@@ -201,6 +201,12 @@ fi
 # Copy data files
 cp -r "$SPARK_HOME/data" "$DISTDIR"
 
+# Make pip package
+echo "Building python distribution package"
+cd $SPARK_HOME/python
+python setup.py sdist
```

Comment:
I think if a user is publishing a release it's reasonable to assume they have Python and at least pandas and numpy. But if this isn't the case working around it with a environment variable same as TGZ sounds like a good suggestions.